ddtrace/tracer/option.go

```diff
@@ -138,19 +138,25 @@ func newConfig(opts ...StartOption) *config {
 		c.version = ver
 	}
 	if v := os.Getenv("DD_TAGS"); v != "" {
-		for _, tag := range strings.Split(v, ",") {
+		sep := " "
+		if strings.Index(v, ",") > -1 {
+			sep = ","
```

Comment:
Let's add a comment above this line
```suggestion
			// falling back to comma as separator
			sep = ","
```